modules/ROOT/pages/8.0-release-notes.adoc

```diff
@@ -126,6 +138,11 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 
 // CCFR here.
 
+=== Split button DOM structure changes
+// #TINY-8665
+
+The internal DOM structure of split buttons has been updated to support the new component architecture. Split button components now use `+.tox-split-button__main+` and `+.tox-split-button__chevron+` CSS classes instead of the previous composite structure. Each component renders as a separate DOM element with independent event handling and focus management. Custom CSS targeting split button internals may require updates to accommodate the new structure.
+
```

Comment:
Suggested changes, also regarding `Custom CSS that targets split button internals may need to be updated to reflect these changes.` what does this look like- will we need to include something in the migration page.
```suggestion
=== Split button DOM structure changes
// #TINY-8665

The internal DOM structure of split buttons has been updated to align with the new component-based architecture. Split button components now use the `+.tox-split-button__main+` and `+.tox-split-button__chevron+` CSS classes in place of the previous composite layout. Each part of the split button is rendered as an independent DOM element with separate event handling and focus management. Custom CSS that targets split button internals may need to be updated to reflect these changes.

```

Reply:
It could be simplified by not mentioning architecture changes. IMO, it should be like this:
```
The split buttons are now rendered as two separate buttons: action and dropdown. Each has its own event handling and focus management. 
```